Index: rcp-web/src/main/java/cn/com/cdboost/collect/consumer/ActivityListener.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- rcp-web/src/main/java/cn/com/cdboost/collect/consumer/ActivityListener.java	(revision 1644)
+++ rcp-web/src/main/java/cn/com/cdboost/collect/consumer/ActivityListener.java	(date 1536291003930)
@@ -10,9 +10,9 @@
 import cn.com.cdboost.collect.service.SmokeDeviceServiceDubbo;
 import cn.com.cdboost.collect.service.SmokeDevlogServiceDubbo;
 import cn.com.cdboost.collect.util.AliyunSmsUtil;
-import cn.com.cdboost.collect.vo.SmokeDeviceSelectTotalInfo;
-import cn.com.cdboost.collect.vo.SmokeDeviceVo;
-import cn.com.cdboost.collect.vo.SmokeDevlogVo;
+import cn.com.cdboost.collec.facadevo.SmokeDeviceSelectTotalInfo;
+import cn.com.cdboost.collec.facadevo.SmokeDeviceVo;
+import cn.com.cdboost.collec.facadevo.SmokeDevlogVo;
 import cn.com.cdboost.collect.vo.alisms.SendSmsParam;
 import com.alibaba.fastjson.JSON;
 import com.google.common.base.Joiner;
